db/objects/V0.1.0.07__foodex_confidence.sql

```diff
@@ -0,0 +1,13 @@
+CREATE TABLE foodex_confidence (
+    id    text   primary key
+);
+
+COMMENT ON TABLE foodex_confidence IS 'Dictionary table of possible values used for indicating how confident we are in our matches to foodex2 values';
+
+
+INSERT INTO foodex_confidence (id)
+VALUES
+    ('Good')
+    , ('Sketchy')
```

Comment:
I think you expected a comment on this one... 

Not convinced that sketchy is necessarily a term we want to use :wink:.  Also high/low may be better than good/bad as a measure of 'our' confidence in the match

```suggestion
    , ('High')
    , ('Medium')
    , ('Low')
```